### PR TITLE
https://lms.ithillel.ua/groups/63c00b312482232c29370c19/homeworks/641…

### PR DESCRIPTION
**Pull request by Pyatnitsa1990**
…e1ce319ed346b863c0986

### CODE REVIEW

File: src/main/java/com/hillel/piatnytsia/homeworks/homework6/duplicates/DuplicateUtils.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public final class DuplicateUtils {
+
+    public static boolean containsDuplicate(int[] nums) {
+        Set<Integer> numbers = new HashSet<>();
+        for (Integer i : nums) {
```
Comment (shaitanaleksander) [inline src/main/java/com/hillel/piatnytsia/homeworks/homework6/duplicates/DuplicateUtils.java]: "i' is a bad var name

File: src/main/java/com/hillel/piatnytsia/homeworks/homework6/duplicates/Main.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.hillel.piatnytsia.homeworks.homework6.duplicates;
+
+public class Main {
+    public static void main(String[] args) {
+        int[] num = {1, 2, 3, 4};
```
Comment (shaitanaleksander) [inline src/main/java/com/hillel/piatnytsia/homeworks/homework6/duplicates/Main.java]: num -> nums

File: src/main/java/com/hillel/piatnytsia/homeworks/homework6/palindrome/PalindromeUtils.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.hillel.piatnytsia.homeworks.homework6.palindrome;
+
+public final class PalindromeUtils {
+    @Override
+    public String toString() {
+        return "Palindrome{}";
+    }
```
Comment (shaitanaleksander) [inline src/main/java/com/hillel/piatnytsia/homeworks/homework6/palindrome/PalindromeUtils.java]: you do not need this method at all

File: src/main/java/com/hillel/piatnytsia/homeworks/homework6/twosum/TwoSumUtils.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+public final class TwoSumUtils {
+
+    public static int[] twoSum(int[] nums, int target) {
+
+        for (int i = 0; i < nums.length; i++) {
+
+            for (int j = i + 1; j < nums.length; j++) {
+                if (nums[i] + nums[j] == target) {
+                    int[] arr = {i, j};
```
Comment (shaitanaleksander) [inline src/main/java/com/hillel/piatnytsia/homeworks/homework6/twosum/TwoSumUtils.java]: arr is a bad name

**Review by shaitanaleksander — APPROVED** [review]

File: src/main/java/com/hillel/piatnytsia/homeworks/homework6/twosum/TwoSumUtils.java (hunk 4 above)
Comment (Pyatnitsa1990) [reply src/main/java/com/hillel/piatnytsia/homeworks/homework6/twosum/TwoSumUtils.java]: I agree

**Pull request merged**